configuration/scripts/icepack_in

```diff
@@ -121,6 +121,7 @@
     atm_data_type   = 'clim'
     ocn_data_type   = 'SHEBA'
     bgc_data_type   = 'default'
+    ice_advc_type   = 'uniform'
```

Comment:
I agree that 'uniform_ice' is better and I've changed it throughout. 

I can think of one particular context where someone *might* want the ability to advect ice in based on input data. On field campaigns (including MOSAiC) for logistical reasons we often do not establish all of our mass balance measurement sites simultaneously. For example, mass balance sites at MOSAiC were installed over a 2.5 month period from late October to early January. There are various ways that one could handle using these observations in the context of single column modeling, one of which involves prescribing ice advection. Essentially, when a new measurement site is added advect in ice of a fractional area that the new site comprises with the properties of the new site while simultaneously advecting out that same fractional area of the grid cell (so that we don't create ridging).

I don't know if anyone is likely to implement something like that anytime soon (or ever) but I think naming this option 'uniform_ice' won't conflict if so.